src/script/interpreter.cpp

```diff
@@ -1126,8 +1166,30 @@ bool TransactionSignatureChecker::CheckSig(const vector<unsigned char>& vchSigIn
     return true;
 }
 
-bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime) const
+bool TransactionSignatureChecker::CheckLockTime(const CScriptNum& nLockTime, bool fSequence) const
 {
+    // Relative lock times are supported by comparing the passed
+    // in lock time to the sequence number of the input. All other
+    // logic is the same, all that differs is what we are comparing
+    // the lock time to.
+    int64_t txToLockTime;
```

Comment:
Since this is repeatedly referred to as a relative lock time,  can we reflect that? `relativeLockTime`?